### PR TITLE
Added instructions for locally downloading YAML

### PR DESCRIPTION
**Pull request by cedricclyburn**
Via #166

### CODE REVIEW

**Comment by erwangranger** [comment]
Thanks for the PR and clarifications it brings. 

That makes sense. We might have to change the PR to merge in "Dev" branch first, and then "main".  
We'll include this into our next set of changes to the materials.

**Comment by cedricclyburn** [comment]
Gotcha, thank you @erwangranger, just let me know and I can move the base. Cheers!

File: bootstrap/applicationset/applicationset-bootstrap.yaml
Comment (guimou) [inline bootstrap/applicationset/applicationset-bootstrap.yaml:1]: Can you change the targetRevision back to dev? We merge into dev first to fully test the deployment, then only to main.

File: content/antora.yml
Comment (guimou) [inline content/antora.yml:1]: use dev branch

File: content/modules/ROOT/pages/05-03-web-app-deploy-application.adoc
Comment (guimou) [inline content/modules/ROOT/pages/05-03-web-app-deploy-application.adoc:1]: dev branch

**Review by guimou — CHANGES_REQUESTED** [review]
Thanks a lot!
As @erwangranger said, the only needed changes are the dev branch instead of main.
I can take care of it if not practical for you.

**Comment by cedricclyburn** [comment]
Thanks for the heads up @guimou! Good luck at Summit, hope to meet you there :)

**Pull request merged**